src/coreclr/jit/optimizer.cpp

```diff
@@ -9407,6 +9407,10 @@ void Compiler::optRemoveRedundantZeroInits()
 
                         break;
                     }
+                    // case GT_CALL:
+                    // TODO: Need to remove redundant zero-inits for "return buffer".
+                    // assert(!"Need to handle zero inits.\n");
+                    // break;
```

Comment:
Hmm, so there is not correctness updated to be had here?

The comment looks a little weird -- is it a `TODO-CQ`?

Reply:
Yes, as far as I know, there is no correctness issue, just some optimization opportunities that I want to handle in a separate PR.